### DISCUSSION TITLE
Default dashboard browse folder

### DISCUSSION THREAD

**Discussion by statdigger** [NetApp/harvest#1770] [General]
Hi all,

In Grafana, when you select Dashboards > Browse you see the three Harvest folders of 7mode, Cluster and General. 
By default, the General folder is expanded. I don't know if this is because it is last in the list.

Is there a way to make the Cluster folder the default expanded folder, or if not, at least not expand the General folder ?.
I appreciate this is a 1st world problem, but as I never use the dashboards in the General folder, it would be nice if the Cluster folder was the default.

Is there a config file I can change to achieve this ?

Thanks.

**Comment by cgrinds** [comment]
hi @statdigger I'm guessing that you are using nabox? I don't see that behavior on standalone Grafana, but do see it with nabox bundled Grafana. Let me see what I can find out. Maybe there is something in Grafana's provisioning config we can tweak.

**Comment by cgrinds** [comment]
hey @ybizeul, @statdigger is using nabox, is this something you are changing in grafana defaults.ini? I ask because a vanilla install of Grafana with imported Harvest dashboards does not auto-expand a folder.

e.g.

<img width="511" alt="image" src="https://user-images.githubusercontent.com/242252/221024421-9088853b-4788-47d3-9219-328c74e9bd10.png">

**Reply by ybizeul** [reply]
It seems your install doesn't have a General folder.
I'm thinking this General folder doesn't appear if all the dashboards you have are in one of the Harvest folders, and if there is anything not in a folder it would be displayed in "General" which would be expanded by default.

**Comment by ybizeul** [comment]
@statdigger You could delete the dashboards from the General folder, you would lose the NAbox dashboard but it looks like you wouldn't miss it, the rest is legacy and has long been moved to each platform subfolder

**Comment by statdigger** [comment]
Many thanks for you feedback so far. To confirm, yes we are using nabox.

I had considered deleting the General dashboards but it seemed like a hammer to crack a nut. I'm happy to do that if it is the only solution but I'm open to other suggestions.

![image](https://user-images.githubusercontent.com/126171182/221125908-cc4a862f-7444-4b61-afdc-f187b38b5238.png)

**Reply by ybizeul** [reply]
Will do some research !
